index.js

```diff
@@ -214,6 +214,14 @@ function showSearch() {
     });
 }
 
+/*
+ * Opens the bookmark menu
+ */
+function showBookmarks() {
+    let activeWindow = windowUtils.getMostRecentBrowserWindow();
+    UITour.showMenu(activeWindow, 'bookmarks');
```

Comment:
Alternatively you can use the above `uiTourHelper` (might be a bit DRYer), your choice.